docs/source/GETTING_STARTED.rst

```diff
@@ -0,0 +1,132 @@
+.. _GETTING_STARTED:
```

Comment:
Maybe this whole file should be a part of the README rather than its own file.  New users want this info, not to have to find the link.

Reply:
I guess this is a personal style thing. Normally my preference is to include usage examples in the README as an entrypoint for new users. However, the kiosk workflow is so complicated that I think it's overwhelming and distracts from the other information in the README. Ultimately I'll defer to your preference, but that's my logic for the decision.

Reply:
Minimal content for starting the kiosk now moved into readme with the bulk of instruction remaining in GETTING_STARTED